can/interfaces/socketcand/socketcand.py

```diff
@@ -231,3 +337,13 @@ def shutdown(self):
         """Stops all active periodic tasks and closes the socket."""
         super().shutdown()
         self.__socket.close()
+
+    @staticmethod
+    def _detect_available_configs() -> List[can.typechecking.AutoDetectedConfig]:
+        try:
+            # Time between beacons no more than 3 seconds. Allow for a little
+            # more than 3s
+            return detect_beacon(3.1)
```

Comment:
```suggestion
            return detect_beacon()
```

Remove the timeout here. This would be a bug anyway, since you expect milliseconds in `detect_beacon`